server/fileupdater.mjs

```diff
@@ -0,0 +1,31 @@
+const VERSION = 2;
+
+export default function FileUpdater(state) {
+  const v = state._meta.version;
+  if(v == VERSION)
+    return state;
+
+  for(const id in state) {
+    const w = state[id];
+
+    if(v < 2 && state[id].type == 'deck' && Array.isArray(w.faceTemplates)) {
+      for(const face of faceTemplates) {
+        if(Array.isArray(face.objects)) {
+          for(const object of face.objects) {
+            if(object.valueType != 'static' && object.value) {
+              if(typeOf object.dynamicProperties != 'object')
```

Comment:
Is that check necessary?

Reply:
Probably not, should not exist in prior versions at all. I just think playing as safe as possible in file updater is a good idea.